Sources/SwifterSwift/UIKit/UICollectionViewExtensions.swift

```diff
@@ -147,6 +147,18 @@ public extension UICollectionView {
         register(UINib(nibName: identifier, bundle: bundle), forCellWithReuseIdentifier: identifier)
     }
 
+    /// SwifterSwift: Safely scroll to possibly invalid IndexPath
+    ///
+    /// - Parameters:
+    ///   - indexPath: Target IndexPath to scroll to
+    ///   - scrollPosition: Scroll position
+    ///   - animated: Whether to animate or not
+    func safeScrollToItem(at indexPath: IndexPath, at scrollPosition: UICollectionView.ScrollPosition, animated: Bool) {
+        guard indexPath.section < numberOfSections && indexPath.section >= 0 else { return }
```

Comment:
This should basically be a copy of `isValidIndexPath`. You can also combine the guards:
```Swift
guard indexPath.section >= 0,
   indexPath.item >= 0,
   indexPath.section < numberOfSections,
   indexPath.item < numberOfItems(inSection: indexPath.section) else {
      return
}
```